Update setup instructions and add Dockerfile (#52)
greenelab/phenoplier: environment/README.md

Keep in mind that the software only runs on Linux or macOS, **Windows is not

> supported** now.

Reviewer: ```suggestion
supported** yet.
```
You could also just leave this last word off, or I would potentially just phrase it as "Windows is not currently supported"

1. Install [Miniconda](https://docs.conda.io/en/latest/miniconda.html) or Anaconda.
1. Open a terminal, run `cd environment` from the folder where you cloned this

>    repo, and execute all following steps in it.

Reviewer: ```suggestion
   repo, and execute all of the following steps there.
```

>     export PYTHONPATH=`readlink -f ../libs/`:$PYTHONPATH

Reviewer 2: I think it's fine to leave as is then. I believe modifying PYTHONPATH is the best alternative to making a package

    ```

    `readlink` might not work on macOS. In that case, simply replace it with
    the absolute path to the `../libs/` folder.

1. Download the data:

    ```bash

>     python scripts/setup_data.py

Author: Good idea, added!
